User.java

```diff
@@ -1,18 +1,20 @@
 
 public class User {
 	/*For the User class the fields needed are:
-	 * Name, age, city, state, dissabilities, and
+	 * Name, age, city, state, disabilities, and
 	 * number of families.
      * 
-     * I added simple setters and getters and some boolean methods 
-     * that decide whether or not the user is a minor
+     * Class also contains multiple methods for creating and sending the
+     * information of the user
+     * 
 	 * */
 	private String name;
 	private int age;
 	private String city;
 	private String state;
 	private String disabilities;
-	private int numOfFamMembers;
+    private int numOfFamMembers;
```

Comment:
Something seems funky about the indentation here, it doesn't quite line up in github. Is it lined up in your editor? If so then this may be a tabs vs spaces issue. Regardless, do you mind fixing it? Over time indentation errors can make skimming the code a lot more challenging.

Reply:
Hey Perry do you know where to edit the indents on cloudshell? I've been looking for it and I can't find it.

Reply:
Hmm not off the top of my head. Cloudshell isn't a commonly used editor, but I think it's actually based on Microsoft's VSCode. Try googling "VSCode change indentation settings" or similar and see if those instructions apply? If not we can hop on a quick call and try to figure it out.